test/JDBC/upgrade/latest/verification_cleanup/14_6/ISC-sequences-vu-verify.mix

```diff
@@ -15,6 +15,9 @@ go
 use isc_sequences_dbone;
 go
 
+select name from sys.schemas order by name;
+go
+
```

Comment:
For the test, please at least consider following scenarios
1. non-MVU case, when user creates a database, there has to be a guest schema in the database, and it is owned by guest user, guest user's default schema is guest schema, which means if guest user creates a table t1, it should be in guest.t1 ([BABEL-2461])
2. Considering version upgrade, after upgrading all existing databases should also reach to the above state
3. During version upgrade, if customer has created a guest user by themself, do nothing, keep it as it is

Reply:
1. This scenario is covered in the non-MVU test [here](https://github.com/babelfish-for-postgresql/babelfish_extensions/pull/1154/files#diff-ac3d2a08c4a0847f26c7470929a33f2a7f88160377aa90e5aa190d67eee1e5c7)

2. I will add this scenario in the mvu test
3. AFAIK we do not do anything with any guest user.